tests/test-sources/plugins/utils/persistence.nix

```diff
@@ -0,0 +1,18 @@
+{
+  empty = {
+    plugins.persistence.enable = true;
+  };
+
+  defaults = {
+    plugins.persistence = {
+      enable = true;
+
+      dir.__raw = ''
+        vim.fn.expand(vim.fn.stdpath("state") .. "/sessions/")
+      '';
```

Comment:
```suggestion
      dir.__raw = "vim.fn.expand(vim.fn.stdpath('state') .. '/sessions/')";
```

Reply:
As you want, I find the one-liner more compact and easier to read but this is useless neat picking.